client/my-sites/site-settings/form-general.jsx

```diff
@@ -170,28 +171,56 @@ class SiteSettingsFormGeneral extends Component {
 		} );
 	};
 
+	renderLanguagePickerNotice = () => {
+		const { fields, translate } = this.props;
+
+		const langId = get( fields, 'lang_id', '' );
+		const matches = /^error_(\w+)$/.exec( langId );
+
+		let notice;
```

Comment:
Nit: if you decide to keep the method as-is, what do you think about renaming `notice` - I expected that it would actually store a `<Notice />`, and not the notice text. So maybe it would be a little more predictable if we rename it to something like `noticeText`.